robot_log_visualizer/ui/autogenerated/visualizer.py

```diff
@@ -239,15 +239,25 @@ def setupUi(self, MainWindow):
         icon = QtGui.QIcon.fromTheme("exit")
         self.actionQuit.setIcon(icon)
         self.actionQuit.setObjectName("actionQuit")
+
+        # Add the GUI components for the open action
```

Comment:
This is an autogenerated file and cannot be touched the correct approach here is to modify the UI file directly with qtcreator

Reply:
Fixed in [be77f98](https://github.com/ami-iit/robot-log-visualizer/pull/80/commits/be77f988f36b1c23a81ba762de8916dab6b1a1c4)